test/extended/image_ecosystem/s2i_ruby.go

```diff
@@ -19,7 +19,7 @@ import (
 var _ = g.Describe("[image_ecosystem][ruby][Slow] hot deploy for openshift ruby image", func() {
 	defer g.GinkgoRecover()
 	var (
-		railsTemplate = "https://raw.githubusercontent.com/openshift/rails-ex/master/openshift/templates/rails-postgresql.json"
+		railsTemplate = "rails-postgresql-example"
```

Comment:
Likewise - is this a 4.x fix that is being backported to 3.11?

Reply:
yes part of 4.x backport to make image-eco work again on 3.11 ... it was broke similar to how 4.x starting breaking cause upstream code repo's did not align with installed templates / imagestreams